.github/workflows/test.yaml

```diff
@@ -134,7 +134,7 @@ jobs:
         id: install
         run: |
           python -um pip install --upgrade wheel
-          python -um pip install --upgrade treelog stringly matplotlib scipy pillow numpy
+          python -um pip install --upgrade treelog stringly matplotlib scipy pillow numpy git+https://github.com/evalf/nutils-SI.git@initialize
```

Comment:
Can you add `nutils-SI` to the container image as well? This seems to work:

```diff
diff --git a/devtools/container/build.py b/devtools/container/build.py
index 04a9c62f..cdd4e323 100644
--- a/devtools/container/build.py
+++ b/devtools/container/build.py
@@ -57,9 +57,10 @@ with ExitStack() as stack:
   if not examples.exists():
     raise SystemExit(f'cannot find example dir: {examples}')

-  container = stack.enter_context(Container.new_from(base, network='none', mounts=[Mount(src=dist, dst='/mnt')]))
+  container = stack.enter_context(Container.new_from(base, mounts=[Mount(src=dist, dst='/mnt')]))

   container.run('pip', 'install', '--no-cache-dir', '--no-index', '--find-links', 'file:///mnt/', 'nutils', env=dict(PYTHONHASHSEED='0'))
+  container.run('pip', 'install', '--no-cache-dir', 'https://github.com/evalf/nutils-SI/archive/initialize.tar.gz', env=dict(PYTHONHASHSEED='0'))
   container.add_label('org.opencontainers.image.url', 'https://github.com/evalf/nutils')
   container.add_label('org.opencontainers.image.source', 'https://github.com/evalf/nutils')
   container.add_label('org.opencontainers.image.authors', 'Evalf')
```

Reply:
fixed